src/test/regress/expected/create_null_dist_key.out

```diff
@@ -670,28 +670,38 @@ SELECT create_distributed_table('sensors_2004', NULL, distribution_type=>null, c
 (1 row)
 
 ALTER TABLE sensors ATTACH PARTITION sensors_2004 FOR VALUES FROM ('2004-01-01') TO ('2005-01-01');
+-- verify we can attach a citus local table
+CREATE TABLE sensors_2005 (measureid integer, eventdatetime date, measure_data jsonb, PRIMARY KEY (measureid, eventdatetime, measure_data));
+SELECT citus_add_local_table_to_metadata('sensors_2005');
+ citus_add_local_table_to_metadata
+---------------------------------------------------------------------
+
+(1 row)
+
+ALTER TABLE sensors ATTACH PARTITION sensors_2005 FOR VALUES FROM ('2005-01-01') TO ('2006-01-01');
```

Comment:
Let's also add a separate test to show that "ATTACH PARTITION" fails when the parent is a citus local table and the child is a distributed table that don't have a shard key.

LGTM othwerwise.